package.json

```diff
@@ -13,12 +13,12 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:grunt build:handlebars",
```

Comment:
I was thinking in regards to CIs, we would want to print all the errors at once (as we do with tests).